src/Search/Search.css

```diff
@@ -0,0 +1,93 @@
+.search-container {
+    background-image: url('../images/vinyl.jpg');
+    height: 100vh;
+    background-position: center;
+    background-repeat: no-repeat;
+    background-attachment: fixed;
+    background-size: cover;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+
+.search-container form {
+    display: flex;
+    flex-direction: column;
+    width: 80%;
+}
+
+.search-container form label {
+    color: #fffaf0;
+    font-size: large;
+    margin-bottom: 8px;
+}
+
+.search-container form div {
+    width: 100%;
+    height: 10vh;
+    display: flex;
+    justify-content: center;
+}
+
+.search-container form div input {
```

Comment:
Did you add the "form, div, input" part? I've never seen that before in CSS, I don't think, I'm curious about it.

Reply:
Yes, it's a way to target a specific group without giving everyone a class name